src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs

```diff
@@ -156,9 +156,9 @@ private string GetMimeMapping(string filePath)
                 {".svg", "image/svg+xml"}
             };
 
-            if (string.IsNullOrEmpty(Path.GetExtension(filePath)))
+            if (string.IsNullOrWhiteSpace(filePath))
             {
-                throw new ArgumentNullException($"Invalid path to file was informed: '{filePath}'");
+                throw new ArgumentNullException("An attempt to get the MIME mapping of an empty path was made.");
```

Comment:
Wrong exception type:

```suggestion
                throw new ArgumentException("An attempt to get the MIME mapping of an empty path was made.");
```

It's not really specific to the mime type though, right? It seems weird to have the detection + exception in this private method in particular, when I'd expect uploading an invalid path to fail anyway.

Reply:
@jcagme :-)